### DISCUSSION TITLE
Logo(image) on readme

### DISCUSSION THREAD

**Discussion by AbduazizZiyodov** [encode/starlette#2797] [Q&A] [answered]
Little stuff about the readme ;)

Readme:
![image](https://github.com/user-attachments/assets/ef884de8-7125-4f03-820c-1142aab58db4)
Web:
![image](https://github.com/user-attachments/assets/9df80e4a-a90b-4562-bf62-4ba4b1ceaf6a)
![image](https://github.com/user-attachments/assets/11914850-f3b7-4638-875e-420293bb8555)

As you can see web shows image according to the applied theme. Github also [supports](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) for showing images according to the current theme(dark/light):

Can we do this for readme also, I did not wanted to open any PR (appearing as a contributor because of changing readme file is kinda strange).

**Comment by Kludex** [comment]
I'd accept a PR for this 🥹👍

**Reply by AbduazizZiyodov** [reply, marked as answer]
#2799 :handshake:

**Reply by Kludex** [reply]
Thanks 🤩
